code/obj/machinery/optable.dm

```diff
@@ -71,6 +72,8 @@
 /obj/machinery/optable/attackby(obj/item/W as obj, mob/user as mob)
 	if (issilicon(user)) return
 	if (istype(W, /obj/item/electronics/scanner)) return // hack
+	if (istype(W, /obj/item/deconstructor)) return //deconstruct_flags
+	if (istype(W, /obj/item/deconstructor/borg)) return //deconstruct_flags
```

Comment:
`istype` already includes child types.

Reply:
Sorry, still very new to this. Fixed up!

Reply:
No worries!